templates/vm.xml.j2

```diff
@@ -1,5 +1,10 @@
 <domain type='{{ libvirt_vm_engine }}'>
   <name>{{ vm.name }}</name>
+  {% if vm.uuid is defined %}
+  <uuid>{{ vm.uuid }}</uuid>
+  {% elif libvirt_vm_default_uuid_deterministic is sameas true or (vm.uuid_deterministic is defined and vm.uuid_deterministic is sameas true) %}
```

Comment:
To evaluate booleans I usually use `libvirt_vm_default_uuid_deterministic | bool`, which supports yes/no etc.

Reply:
I believe the syntax `x is sameas true` also supports yes/no. But if you want the other syntax for more code consistency I'll do it.

Reply:
Looks like it doesn't accept strings, but is more like the Python `is` keyword:

```
mark@mark-xps15:~$ ansible localhost -m debug -a 'msg={{ "yes" is sameas true }}'
localhost | SUCCESS => {
    "msg": false
}
mark@mark-xps15:~$ ansible localhost -m debug -a 'msg={{ "true" is sameas true }}'
localhost | SUCCESS => {
    "msg": false
}
mark@mark-xps15:~$ ansible localhost -m debug -a 'msg={{ true is sameas true }}'
localhost | SUCCESS => {
    "msg": true
}
mark@mark-xps15:~$ ansible localhost -m debug -a 'msg={{ 'no' is sameas false }}'
localhost | SUCCESS => {
    "msg": false
}
mark@mark-x
```